src/AnnouncementBubble/NormalAnnouncementBubble.tsx

```diff
@@ -33,14 +37,21 @@ export const NormalAnnouncementBubble: React.FC<Props> = ({
   className,
   onDelete,
   onReply,
+  readBy,
+  recipientType,
   repliesDisabledMsg,
   replyButtonText,
   senderIcon,
   senderName,
   sentAtTimestamp,
+  userType,
 }: Props) => {
   const deleteMenu = formDeleteMenu(onDelete);
   const replyButton = formReplyButton(onReply, repliesDisabledMsg, replyButtonText);
+  const readReceiptsTooltip =
+    userType === "teacher" && readBy && readBy.length > 0
```

Comment:
We could use optional chaining when checking `readBy`. -> `readBy?.length >0`

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html